renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -799,7 +799,9 @@ void GLRenderState::FetchState(WrappedOpenGL *gl)
     {
       GLboolean layered = GL_FALSE;
 
-      m_Real->glGetIntegeri_v(eGL_IMAGE_BINDING_NAME, i, (GLint *)&Images[i].res.name);
+      GLint name;
+      m_Real->glGetIntegeri_v(eGL_IMAGE_BINDING_NAME, i, &name);
+      Images[i].res = TextureRes(ctx, name);
```

Comment:
I'd prefer to set these namespaces up-front in `GLRenderState` with the others, for consistency if nothing else. Looks like I just forgot this array when I wrote that for some reason.